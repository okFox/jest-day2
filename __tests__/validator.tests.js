const Validator = require('../lib/Validator');

describe('Validator', () => {
    beforeAll(() => {
        validator = 
    })
})