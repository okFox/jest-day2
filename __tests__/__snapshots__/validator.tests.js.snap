// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Validator all fields required returns the field 1`] = `"age is required"`;
